docker-compose-dev.yml

```diff
@@ -12,29 +13,27 @@ services:
       - 8080:8080
       - 5686:5601
     environment:
-      - FORM_RUNNER_INTERNAL_HOST=http://form-runner:3009
+      - FORM_RUNNER_INTERNAL_HOST=http://form-runner-dc:3009
       - FORM_RUNNER_EXTERNAL_HOST=http://localhost:3009
-      - DATABASE_URL=postgresql://postgres:password@fab-db:5432/fund_builder   # pragma: allowlist secret
+      - DATABASE_URL=postgresql://postgres:password@fab-db-dc:5432/fund_builder   # pragma: allowlist secret
       - FLASK_DEBUG=0
       - FLASK_ENV=development
       - SECRET_KEY=local
-    depends_on: [fab-db, form-runner]
+    depends_on: [fab-db-dc, form-runner-dc]
 
-  fab-db:
+  fab-db-dc:
     image: postgres
-    container_name: fab-db
+    container_name: fab-db-dc
     ports:
-      - 5432:5432
+      - 5435:5432
     volumes:
       - ~/apps/postgres:/var/lib/postgresql/data
     environment:
       - POSTGRES_PASSWORD=password
       - POSTGRES_DB=fund_builder
-      - DATABASE_URL=postgresql://postgres:password@fab-db:5432/fund_builder
-      - DATABASE_URL_UNIT_TEST=postgresql://postgres:password@fab-db:5432/fab_unit_test
 
-  form-runner:
-    container_name: form-runner
+  form-runner-dc:
+    container_name: form-runner-dc
```

Comment:
bit curious about the dc post fix ?

Reply:
dc=docker-compose - I just needed to separate it from the ones in the dev container... (but that was already merged)